src/test/java/org/folio/innreach/controller/d2ir/PatronInfoControllerTest.java

```diff
@@ -304,6 +307,99 @@ void return200HttpCode_and_patronInfoResponseWithPatronInfo_hasIgnoreCaseCorrect
     assertNotNull(response.getPatronInfo());
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = {"jo hn do e","do e jo hn","jo hn do e pauL","do e Jo hn pauL","do e, Jo hn, pauL", "do e Jo hn, pauL"})
```

Comment:
I'm thinking that perhaps it would make this test easier to understand can we make the names more like they would be in real life instead of cutting them up. Like maybe do `john jimmy doe smith` for example. Two common first names and two common last names.

Reply:
Common names in my part of the world anyway.  :)

Reply:
Yes, agreed. It will make more sense to have a descriptive names. 
Changed the name as per the suggestion